configure.ac

```diff
@@ -1592,6 +1592,9 @@ pre_PKG_CONFIG_CFLAGS=$CFLAGS
 pre_PKG_CONFIG_LIBS=$LIBS
 
 PKG_PROG_PKG_CONFIG(0.9.0)
```

Comment:
Perfect. This is a small change that the Emacs folk might be interested in as well.